src/components/PostsList/PostsList.jsx

```diff
@@ -1,37 +1,50 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './PostsList.scss';
 
-export const PostsList = () => (
-  <div className="PostsList">
-    <h2>Posts:</h2>
+export const PostsList = ({ select, posts, showPostInfo, postSelected }) => {
+  let list = posts;
 
-    <ul className="PostsList__list">
-      <li className="PostsList__item">
-        <div>
-          <b>[User #1]: </b>
-          sunt aut facere repellat provident occaecati excepturi optio
-        </div>
-        <button
-          type="button"
-          className="PostsList__button button"
-        >
-          Close
-        </button>
-      </li>
+  if (select !== '0') {
+    list = posts.filter(post => post.userId === +select);
+  }
```

Comment:
consider to use `useMemo` for this purpose

Reply:
done